code/modules/projectiles/projectile/bullets.dm

```diff
@@ -232,3 +232,6 @@
 	if(isalien(target))
 		return 0
 	..() // Execute the rest of the code.
+
+/obj/item/projectile/bullet/a762

```

Comment:
Just use midbullet2, oddly enough we have a762 magazines, yet some seem to use the base bullet type, which does 60 damage. midbullet2 is the exact same as this, 25 damage and no other effects.